src/App.tsx

```diff
@@ -1,50 +1,115 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+
+type Props = {
+  initialGoods: string[];
+};
+
+enum SortType {
+  Alphabetical = 'Alphabetical',
+  ByLength = 'ByLength',
+  None = 'None',
+}
+
+export const App: React.FC<Props> = ({ initialGoods }) => {
+  const [goods, setGoods] = useState([...initialGoods]);
```

Comment:
You can just use pass props like it is
```suggestion
  const [goods, setGoods] = useState(initialGoods);
```